_posts/2023-7-12-Add-client-side-HTTP-authentication-mechanism-support.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "Upcoming client side HTTP authentication mechanisms support"
+date: 2023-07-12
+tags: http authentication client
+synopsis: Implementation of client side http authentication mechanism for Elytron.
```

Comment:
Just a minor, s/http/HTTP.

You can say something like "The implementation of client side HTTP authentication mechanisms in Elytron is in progress."